bookwyrm/templates/settings/users/user_moderation_actions.html

```diff
@@ -49,7 +49,7 @@ <h3>{% trans "User Actions" %}</h3>
 
         {% if user.local %}
         <div>
-            <form name="permission" method="post" action="{% url 'settings-user' user.id %}">
+            <form name="permission" method="post" action="{% url 'settings-user' user.id report.id %}">
```

Comment:
The `UserAdmin` view doesn't expect a `report_id` keyword so trying to change user permissions throws a `TypeError`.

I'm guessing this is supposed to have a check for the the presence of a `report_id` kwarg and then run a `models.Report.record_action()`?